platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -50,6 +50,33 @@ Ant has been updated to https://archive.apache.org/dist/ant/RELEASE-NOTES-1.10.1
 
 === Deprecations
 
+[[minimum_daemon_jvm_version]]
+==== Deprecated running the Gradle daemon with JVM versions 16 and earlier
+
+Starting in Gradle 9.0, the Gradle daemon will require JVM 17 or later to run and Gradle API classes will be compiled to target JVM 17 bytecode.
```

Comment:
I would start general and then refine what we mean further down.  That means instead of saying "Gradle daemon", we'll just say "Gradle". I think otherwise, people might think that if they avoid using the Gradle daemon, they're not affected.

```suggestion
Starting in Gradle 9.0, Gradle will require JVM 17 or later to run. Most Gradle APIs will be compiled to target JVM 17 bytecode.
```